quickwit/quickwit-cluster/src/member.rs

```diff
@@ -20,18 +20,21 @@
 use std::collections::HashSet;
 use std::net::SocketAddr;
 
+use anyhow::anyhow;
 use chitchat::{ClusterStateSnapshot, NodeId, NodeState};
 use itertools::Itertools;
 use quickwit_proto::indexing_api::IndexingTask;
-use serde::{Deserialize, Serialize};
 use tracing::{error, warn};
 
 use crate::QuickwitService;
 
 // Keys used to store member's data in chitchat state.
 pub(crate) const GRPC_ADVERTISE_ADDR_KEY: &str = "grpc_advertise_addr";
 pub(crate) const ENABLED_SERVICES_KEY: &str = "enabled_services";
-pub(crate) const RUNNING_INDEXING_PLAN: &str = "running_indexing_plan";
+// An indexing task key is formatted as
+// `{INDEXING_TASK_PREFIX}{INDEXING_TASK_SEPARATOR}{index_id}{INDEXING_TASK_SEPARATOR}{source_id}`.
+pub(crate) const INDEXING_TASK_PREFIX: &str = "indexing_task";
+pub(crate) const INDEXING_TASK_SEPARATOR: &str = ":";
```

Comment:
I put ":" at first but we may relax again the field name validation rules... So we may want to choose something else.